Exercise_A/index.html

```diff
@@ -17,29 +18,227 @@
+            <div class="image-overlay">
+              <div class="image-title fs-6">THREE THING</div>
+              <div class="image-description fs-4">This</div>
+            </div>
+          </div>
+        </div>
+        <div class="col-lg-3 col-sm-6 g-0">
+          <div class="captionbox">
+            <img class="image-img" src="assets/images/4.jpg" alt="">
+            <div class="image-overlay">
+              <div class="image-title fs-6">FOUR THING</div>
+              <div class="image-description fs-4">Bullshit</div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
 
-    <!-- All scripts we need jquery, bootstrap, smooth scroll, local script -->
-
-    <!-- 
+  <section id="smallbox">
+    <div class="bg-dark text-center">
+      <h2 class="title-1">Seriously, It's This Fucking Template !</h2>
+      <a class="second-btn btn btn-primary btn-lg rounded-pill wow tada" href="#" role="button">THAT'S AMAZING!</a>
+    </div>
+  </section>
+  <!-- contact me container -->
+  <section id="contact">
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-8 offset-lg-2 text-center">
+          <h2 class="title-2">My Contact Information</h2>
+          <hr class="hr-1 my-4 mx-auto">
+          <p class="contact-paragraph">Here's my phone number and email address, because hey that's part of the template
+            and why would you not
+            include that on your webpage? You'd be a fool not to! I can't wait for all the job offers to come pouring in
+            from all the companies wanting my insane website design skills.</p>
+          <p class="contact-paragraph">Honestly this template does look really nice, though.</p>
+        </div>
+        <div class="col-lg-4 offset-lg-2 text-center">
+          <i class="fas fa-phone-alt fa-3x wow bounceIn "></i>
+          <p class="contact-phone">+1-608-317-1888</p>
+        </div>
+        <div class="col-lg-4 text-center">
+          <i class="far fa-envelope fa-3x wow bounceIn "></i>
+          <p class="contact-email"><a href="mailto:abc@gmail.com">holy@shit.com</a></p>
+        </div>
+      </div>
+    </div>
+  </section>
+  <!-- All scripts we need jquery, bootstrap, smooth scroll, local script -->
+  <!-- Bootstrap JS -->
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-U1DAWAznBHeqEIlVSCgzq+c9gqGAJn5c/t99JyeKa9xxaYpSvHU5awsuZVVFIhvj"
+    crossorigin="anonymous"></script>
+  <!-- WOW JS -->
+  <script src="assets/js/wow.js"></script>
+  <script>
+    new WOW().init();
+  </script>
+  <!-- Local JS -->
+  <script src="assets/js/script.js"></script>
+  <!-- 
       All of your script tags for your JavaScript Bootstrap components, refer to the Practice folder
 
       If you do not use CSS to do smooth scroll then... You need the CDN for Smooth Scroll
       Your custom JavaScript
      -->
-  </body>
-</html>
+</body>
+
+</html>
```

Comment:
Bravo! You have nailed the cloning of the website. It shows that you have excellent knowledge of HTML, CSS, Bootstrap, DOM, and JavaScript. Keep up the good work, and keep on grinding.